lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java

```diff
@@ -81,37 +78,31 @@ public class CheckPassportHandler
     // Shared DataStore (Write)
     private final DataStore<DocumentCheckResultItem> documentCheckResultStore;
 
-    public CheckPassportHandler() {
-        ServiceFactory serviceFactory = new ServiceFactory();
-
-        this.objectMapper = serviceFactory.getObjectMapper();
-        this.passportConfigurationService = serviceFactory.getPassportConfigurationService();
-
-        this.eventProbe = serviceFactory.getEventProbe();
-        this.sessionService = serviceFactory.getSessionService();
-        this.personIdentityService = serviceFactory.getPersonIdentityService();
-
-        // Service is internal to CheckPassportHandler
-        this.documentDataVerificationService =
-                constructDocumentDataVerificationService(serviceFactory);
+    private final ThirdPartyAPIServiceFactory thirdPartyAPIServiceFactory;
 
-        this.documentCheckResultStore = serviceFactory.getDocumentCheckResultStore();
+    public CheckPassportHandler() {
+        // DocumentDataVerificationService is internal to CheckPassportHandler
+        this(
+                new DocumentDataVerificationService(
```

Comment:
This is the main constructor used at runtime and only sets up the DocumentDataVerificationService. The second constructor does the remaining setup, to avoid duplication and increase test coverage of all the setup being done in the constructor.

The second constructor allows mocking out DocumentDataVerificationService for testing, with out needing it created. 

https://github.com/alphagov/di-ipv-cri-uk-passport-api/blob/19c2eea82f787e56c6530d3fe08d93f4824a013c/lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java#L94


https://github.com/alphagov/di-ipv-cri-uk-passport-api/blob/19c2eea82f787e56c6530d3fe08d93f4824a013c/lambdas/checkpassport/src/test/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandlerTest.java#L115